intervals/main_objs.py

```diff
@@ -888,6 +896,146 @@ def distance(self, df=None, n=3):
     def getDistance(self, df=None, n=3):
         return self.distance(df, n)
 
+    # July 2022 helper for flexed entries
+    def _flexed_sum(self, item, head_flex):
+        if item[0] <= head_flex:
+            item[0] = 0
+        return sum(item)
+
+    # July 2022 helper for flexed entries updated
+    def flexed_distance(self, head_flex, df=None, n=3):
+          '''
```

Comment:
There are also two versions of flexed_distance that appear to be identical. Let's delete one.

Reply:
Done